src/data-table/_data-table.scss

```diff
@@ -24,6 +24,7 @@
   white-space: nowrap;
   font-size: 13px;
   background-color: unquote("rgb(#{$color-white})");
+  -webkit-transform: translate3d(0,0,0);
```

Comment:
We don't prefix anything in source. Please use standard rules only. Prefixing is fine automatically in the build process.

Amend this commit so no extra commits are added to the history please.